core/ui.js

```diff
@@ -110,7 +110,7 @@ function loadDeletedMsgTag(currentNode)
         request.onsuccess = (e) =>
         {
             messageText.textContent = ""
-            const textStyle =  localStorage["theme"] == "\"dark\"" ? "font-style: normal; color: rgba(241, 241, 242, 0.95)" : "font-style: normal; color: rgba(48, 48, 48)"
+            const textStyle =  localStorage["theme"] == "\"dark\"" ? "font-style: normal; color: rgba(241, 241, 242, 0.95)" : "font-style: normal; color: rgba(128, 128, 128)"
```

Comment:
Actually I meant that it'll be better if only the "Restored message" text will be gray, and the message itself will be black. This way it'll be easier to distinguish.

Reply:
Done ^^